defaults/test_default.yaml

```diff
@@ -1,3 +1,9 @@
+# NOTE: set the 'tablets_initial_scale_factor' explicitly while the following
+#       Scylla change is not merged and backported to older scylla branches.
+#       https://github.com/scylladb/scylladb/pull/22522
```

Comment:
Note: This will never not get backported to older branches, but once this drops, we should remove this either way